packages/core/strapi/package.json

```diff
@@ -178,7 +178,7 @@
     "yup": "0.32.9"
   },
   "devDependencies": {
-    "@strapi/pack-up": "workspace:*",
+    "@strapi/pack-up": "4.23.0",
```

Comment:
Does it need to be listed as both a dev and regular dependency here?

Reply:
no it doesn't, nice spot